docs/sources/whatsnew/whats-new-in-v7-3.md

```diff
@@ -46,10 +46,52 @@ These features are included in the Grafana Enterprise edition software.
 
 ### Auditing
 
+Auditing tracks important changes to your Grafana instance to help you manage and mitigate suspicious activity and meet compliance requirements. Grafana logs events (as JSON) to file or directly to [loki](/oss/loki/).
+
+Example of a login event:
+
+```json
+{
+  "timestamp": "2020-10-22T10:18:00.838094347Z",
+  "user": {
+    "userId": 1,
+    "orgId": 1,
+    "isAnonymous": false
+  },
+  "action": "login-grafana",
+  "result": {
+    "statusType": "success",
+    "statusCode": 200
+  },
+  "requestUri": "/login",
+  "ipAddress": "127.0.0.1:41324",
+  "userAgent": "Chrome/86.0.4240.111",
+  "grafanaVersion": "7.3.0"
+}
+``` 
+
+For more details, see the [Auditing docs](/).
+
 ### Datasource Usage Insights
 
+Datasource usage insights is a new feature in Grafana Enterprise 7.3 that allows you to gain insight into how a datasource is being used and how well it works. There is a new tab in the datasource settings page called insights that will show you information about how the datasource has been used in the past 30 days.
+
+Insights per day:
```

Comment:
Not sure I understand this part, do we have the insights 👇  for the last 30 days or is there a count looking 30 days back every day?